docs/docs/rest-api/public/api/v2/types/newMetrics.raml

```diff
@@ -1,18 +1,18 @@
 #%RAML 1.0 Library
 types:
-  DropwizardCounter:
```

Comment:
do we have to have the prefix New? Why not just Meter, Counter, Gauge... ?

Reply:
➕ 1

Reply:
What's the motivation for renaming these? I think I would prefer DropwardCounter and KamonCounter over OldCounter and NewCounter.

Reply:
(or, drop the `New`)

Reply:
I think it was motivated by the fact that it was suggested to have library independent API types (finally) instead of one directly connected to the implementation. So I guess in the end it boiled down to renaming... I actually like the change.

Reply:
Alena and Ken suggested not to expose implementation details via RAML types. I won't be able to get rid of `New` everywhere though. Is it fine with you?

Reply:
Guys?

Reply:
where is it not possible to drop new?

Reply:
Because
```
#%RAML 1.0 Library
uses:
  newMetrics: newMetrics.raml
  deprecatedMetrics: deprecatedMetrics.raml
types:
  Metrics: deprecatedMetrics.DeprecatedMetrics | newMetrics.NewMetrics
```
We can't have two `metrics.raml` files, and two classed both named`Metrics`.